src/shared/test/has-own-test.js

```diff
@@ -0,0 +1,14 @@
+import { hasOwn } from '../has-own';
+
+describe('hasOwn', () => {
+  [
+    [{ foo: 'bar' }, 'foo', true],
+    [{ foo: 'bar' }, 'bar', false],
+    [Object.create(null), 'foo', false],
```

Comment:
Wondering if a test like

`[Object.create(null), 'hasOwnProperty', false]`

would be illustrative.

Reply:
`[{}, 'hasOwnProperty', false]` would be useful actually, since this is an important difference between `hasOwn({}, 'hasOwnProperty')` and `'hasOwnProperty' in {}`.